### PR TITLE
(Mostly) Automate data migration fixture import process

### PR DESCRIPTION
**Pull request by bess**
Also, improve documentation of the process.

Work toward:
* #925

### CODE REVIEW

File: docs/test_data_in_production.md [hunk 1]
```diff
@@ -19,57 +19,22 @@ Work with RDOS to ensure we're identifying representative samples. We want to co
...
\ No newline at end of file
+  1. Regenerate the json representations of the migration data by running:
+     ```
+     DATA_MIGRATION=true bundle exec rspec spec/system/data_migration
+     ```
+
+  2. Copy the newly created .json files to the server where you want to load them: 
+    
+    `scp tmp/data_migration/*.json deploy@p
+dc_describe_staging1.princeton.edu:/tmp`
```
Comment (mccalluc) [inline docs/test_data_in_production.md:30]:
```suggestion
    ```
    scp tmp/data_migration/*.json deploy@pdc_describe_staging1.princeton.edu:/tmp
    ```
```

File: docs/test_data_in_production.md [hunk 2]
```diff
@@ -19,57 +19,22 @@ Work with RDOS to ensure we're identifying representative samples. We want to co
...
+
+1. Then, ssh to the server where you want to create this sample work as the `deploy` user. 
+2. Run the import rake task, specifying the location of the .json files and the netid of the user they should import as:
```
Comment (mccalluc) [inline docs/test_data_in_production.md:33]:
```suggestion
3. Then, ssh to the server where you want to create this sample work as the `deploy` user. 
4. Run the import rake task, specifying the location of the .json files and the netid of the user they should import as:
```

File: docs/test_data_in_production.md [hunk 3]
```diff
@@ -19,57 +19,22 @@ Work with RDOS to ensure we're identifying representative samples. We want to co
...
+  ```
+  bundle exec rake works:import_works\[/path/to/json/files,bs3097]
```
Comment (mccalluc) [inline docs/test_data_in_production.md:35]:
```suggestion
  bundle exec rake works:import_works\[/tmp,bs3097]
```

In the previous step, I think you identify a destination, so I think you could use that same destination here... Except there may be other files you don't want in `/tmp` ... maybe a date-stamped subdirectory just for this transfer? Or maybe the script will skip files that aren't json?

**Review by mccalluc — COMMENTED** [review]
Markdown formatting issue: I think suggestion will fix it.

I may be missing something but could there already be other files in `tmp` before you `scp`?

File: docs/test_data_in_production.md (hunk 3 above)
Comment (Twade968) [reply docs/test_data_in_production.md:35]: The JSON files are not going to /tmp. They are going to a tmp/data_migration directory within the application. When they are copied to the server, they will go wherever the user decides to put them.

**Review by Twade968 — APPROVED** [review]

**Pull request merged**